pkg/varlinkapi/containers.go

```diff
@@ -100,24 +102,33 @@ func (i *LibpodAPI) ListContainerProcesses(call ioprojectatomicpodman.VarlinkCal
 	if err != nil {
 		return call.ReplyContainerNotFound(name)
 	}
-	containerState, err := ctr.State()
-	if err != nil {
-		return call.ReplyErrorOccurred(err.Error())
-	}
-	if containerState != libpod.ContainerStateRunning {
-		return call.ReplyErrorOccurred(fmt.Sprintf("container %s is not running", name))
+	if call.WantsMore() {
+		call.Continues = true
 	}
-	var psArgs []string
-	psOpts := []string{"-o", "uid,pid,ppid,c,stime,tname,time,cmd"}
-	if len(opts) > 1 {
-		psOpts = opts
-	}
-	psArgs = append(psArgs, psOpts...)
-	psOutput, err := ctr.GetContainerPidInformation(psArgs)
-	if err != nil {
-		return call.ReplyErrorOccurred(err.Error())
+	for {
+		containerState, err := ctr.State()
+		if err != nil {
+			return call.ReplyErrorOccurred(err.Error())
+		}
+		if containerState != libpod.ContainerStateRunning {
+			return call.ReplyErrorOccurred(fmt.Sprintf("container %s is not running", name))
+		}
+		var psArgs []string
+		psOpts := []string{"-o", "uid,pid,ppid,c,stime,tname,time,cmd"}
+		if len(opts) > 1 {
+			psOpts = opts
+		}
+		psArgs = append(psArgs, psOpts...)
+		psOutput, err := ctr.GetContainerPidInformation(psArgs)
+		if err != nil {
+			return call.ReplyErrorOccurred(err.Error())
+		}
+		if !call.WantsMore() {
+			return call.ReplyListContainerProcesses(psOutput)
+		}
+		call.ReplyListContainerProcesses(psOutput)
+		time.Sleep(1 * time.Second)
```

Comment:
Worth making this configurable via the varlink API so users can specify how fast they want things refreshing?

Reply:
I vote no.  Because then the func has to be expanded to take an interval even if the interval isn't used.  IMHO, if they want a 2+ second interval, they can implement that on the client side as it should be slow enough.

Reply:
How about you configure it in podman varlink --refreshtime